easybuild/easyconfigs/m/Multiwfn/Multiwfn-3.8_dev-intel-compilers-2022.1.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MakeCp'
+
+name = 'Multiwfn'
+version = '3.8_dev'
+
+homepage = 'http://sobereva.com/multiwfn/'
+description = """Multiwfn is an extremely powerful program for realizingi
+ electronic wavefunction analysis, which is a key ingredient of quantum
+ chemistry. Multiwfn is free, open-source, high-efficient, very user-friendly
+ and flexible, it supports almost all of the most important wavefunction
+ analysis methods."""
+
+toolchain = {'name': 'intel-compilers', 'version': '2022.1.0'}
```

Comment:
This should be in the iimkl/2022a toolchain
Then you can drop the dependency on imkl.